spec/lib/query/query_supporters_spec.rb

```diff
@@ -237,8 +238,8 @@
         supporter1.save!
       }
 
-      it 'finds when using character filled phone number' do 
-        result = QuerySupporters.full_search(np.id, { search: "A search term" })
+      it 'finds when using character filled phone number' do
+        result = QuerySupporters.full_search(np.id, { search: "search term" }) # if this starts with A it will catch Cacau is that bad? I don't know
```

Comment:
I think it doesn't make sense to catch Cacau just because it starts with an "A". It is treating like as if we were searching only for "A".

Reply:
> I think it doesn't make sense to catch Cacau just because it starts with an "A". It is treating like as if we were searching only for "A".

I agree but I'm not quite sure on the solution.